### PR TITLE
hotfix-extrap-house

### PR DESCRIPTION
**Pull request by jjcherian**
## Description

Hot fixes for the extrapolation step + using the presidential margins to infer a ticket splitting estimate in each house / senate race.

### CODE REVIEW

File: src/elexmodel/models/BootstrapElectionModel.py [hunk 1]
```diff
@@ -61,6 +61,10 @@ def __init__(self, model_settings={}, versioned_data_handler=None):
             "agg_model_hard_threshold", True
         )  # use sigmoid or hard thresold when calculating agg model
         self.district_election = model_settings.get("district_election", False)
+        self.office = model_settings.get("office", None)  # office of the election
+        self.election_id = model_settings.get("election_id", None)  # election ID
+        self.geographic_unit_type = model_settings.get("geographic_unit_type", None)  # geographic unit type
```
Comment (dmnapolitano) [inline src/elexmodel/models/BootstrapElectionModel.py]: Where are these three fields used? 🤔

Comment (jjcherian) [reply src/elexmodel/models/BootstrapElectionModel.py]: Oops dead code

Comment (dmnapolitano) [reply src/elexmodel/models/BootstrapElectionModel.py]: No worries! 😄

**Review by dmnapolitano — APPROVED** [review]
Thanks, this is cool!  I wonder if, given this and your comment the other day about how our Senate baseline should be the same as the Presidential one, we should consider a hierarchical model in the future 👀

**Review by lennybronner — APPROVED** [review]

**Pull request merged**